libs/service-portal/core/src/lib/messages.ts

```diff
@@ -841,4 +841,41 @@ export const m = defineMessages({
     defaultMessage: 'Endurnýja',
     description: 'Renew',
   },
+  chooseDelegation: {
```

Comment:
Should all of these be in the access control module instead of in `service-portal-core`?

Reply:
Yes good point